pkg/operator/status.go

```diff
@@ -254,6 +255,18 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		Status: configv1.ConditionTrue,
 		Reason: asExpectedReason,
 	}
+
+	allReady, notReadyPools, err := optr.arePoolsReadyToUpgrade()
+	if err != nil {
+		return err
+	}
+	if allReady {
+		return optr.updateStatus(co, coStatus)
+	}
+	coStatus.Status = configv1.ConditionFalse
+	coStatus.Reason = "PoolsNotReadyToUpgrade" // FIXME(runcom)
+	coStatus.Message = fmt.Sprintf("The following pools are still not ready to upgrade, please check them out: %s", strings.Join(notReadyPools, ","))
```

Comment:
Instead of `please check them out` I think it'd be more valuable to make it a bit more precise, since maybe not all sysadmin would know what a `pool` is, maybe specify: `please check the MachineConfigPool: %v with: oc describe mcp/${pool}` for which pool, etc.

Mostly because the CVO states:
`An operator reports Upgradeable as false when it wishes to prevent an upgrade for an admin-correctable condition. The component should include a message that describes what must be fixed.`

So maybe its worth it to bubble up the error here too? So the CVO can report it.

Reply:
Sure, please check them out was for testing as I want to see that before polishing this out and addressing all questions :joy: